examples/chart/event-handler/templates/configmap.yaml

```diff
@@ -11,6 +11,11 @@ data:
     batch = {{ .Values.eventHandler.batch }}
     window-size = {{ default "24h" .Values.eventHandler.windowSize | quote }}
     debug = {{ default "false" .Values.eventHandler.debug }}
+    {{- if .Values.eventHandler.types }}
+    types = "{{ .Values.eventHandler.types }}"
+    {{- end }}
+    skip-event-types = "{{ default "" .Values.eventHandler.skipEventTypes }}"
+    skip-session-types = "{{ default "" .Values.eventHandler.skipSessionTypes }}"
```

Comment:
The `skip-session-types` default of "" would be very noisy. When unset, the default is `"print"`.

I'd recommend leaving these unset in the config file if unset in the helm chart values. That way the event handler's own default unset behaviors can take effect.